examples/simple.rs

```diff
@@ -0,0 +1,46 @@
+//! Simple download example.
+//!
+//! Run with
+//!
+//! ```not_rust
+//! cargo run --example simple
+//! ```
+//!
+use color_eyre::{eyre::Report, Result};
+use std::path::PathBuf;
+use tracing_subscriber;
+use tracing_subscriber::EnvFilter;
+use trauma::{download::Download, downloader::DownloaderBuilder};
+// use opentelemetry::sdk::export::trace::stdout;
+// use tracing_subscriber::layer::SubscriberExt;
+// use tracing_subscriber::Registry;
+
+#[tokio::main]
+async fn main() -> Result<(), Report> {
+    // Setup the application.
+    color_eyre::install()?;
+
+    // Prepare the logger.
+    // Currently commented out due to a bug with the reqwest-tracing crate:
+    // https://github.com/TrueLayer/reqwest-middleware/issues/35
+    // let tracer = stdout::new_pipeline().install_simple();
+    // let telemetry = tracing_opentelemetry::layer().with_tracer(tracer);
+    // let subscriber = Registry::default().with(telemetry);
+    // tracing::subscriber::set_global_default(subscriber).unwrap();
+    //
+    // Falling back to the regular tracing library.
+    std::env::set_var("RUST_LOG", "debug");
+    tracing_subscriber::fmt::fmt()
+        .with_env_filter(EnvFilter::from_default_env())
```

Comment:
You can use [`tracing::Level`](https://github.com/tokio-rs/tracing/blob/846a53e3972a7f5d53358095afe6a70507c99b03/examples/examples/fmt.rs#L6-L8) here

Reply:
Addressed ✅